src/Module.php

```diff
@@ -133,6 +133,27 @@ class Module extends \yii\base\Module implements BootstrapInterface
      */
     public $traceLine = self::DEFAULT_IDE_TRACELINE;
 
+    /**
+     * @var string|array HTML title
+     * ```php
+     *  ...
+     *  'html_title'=> 'Custom Debug Title',
+     *  ...
+     * ```
+     * ```php
+     * 'html_title'=> [
+     *     'base-url'=> 'page title',
+     *     'http://qa.example.test'=> 'qa debugger'
+     * ]
+     * ```
+     */
+    public $html_title;
```

Comment:
pageTitle

Reply:
Can't we make it string|callable instead? What if I have an application supporting 100s of sub-domains?

```php
'pageTitle' => function ($url) {
   $domain = getDomain($url);
   return $domain . ' debugger';
}
```

Reply:
Agreed, callable will be more flexible than an array